rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2022-09-08"
+channel = "stable"
```

Comment:
@FlorentinDUBOIS Now it means the latest stable version. I think it's OK for now and we may choose an MSRV when #224 finished and we think we've completed the major feature set.